### PR TITLE
Refactored DB connection into a single file inc/db.php

### PR DESCRIPTION
**Pull request by lionello**

### CODE REVIEW

**Comment by nihaopaul** [comment]
sorry dude, this has conflicts

**Comment by lionello** [comment]
Rebased

**Comment by lionello** [comment]
Sorry, rebased onto master and just saw there was another commit there 61741bc. Let me know if you want me to remove it.

File: api.php [hunk 1]
```diff
@@ -7,9 +8,6 @@
 $salt = 'salT';
 $passwordx = sprintf("%08x", crc32($salt.strtoupper($email)));
 
-// add SetEnv MYSQL_PASSWORD "blah" to this site's Apache conf
-$link = mysql_connect('localhost', 'webuser', getenv('MYSQL_PASSWORD'))
-	or die('mysql_connect error');
 $email2 = '"'.mysql_real_escape_string($email, $link).'"';
 $password2 = '"'.mysql_real_escape_string($password, $link).'"';
 mysql_query("UPDATE members.Users SET count = count + 1 WHERE CURDATE() <= paid AND email = $email2 AND password = $password2", $link)
```
Comment (nihaopaul) [inline api.php:13]: why not prepare the queries with "sprintf" then you dont need to worry about turning into strings.. return sprintf("there are %n monkeys.. that smell like %s", 12, 'poop').. will still merge, but think about cleaning up

**Pull request merged**